fe/lib/components/LocationSuggest/LocationSuggest.tsx

```diff
@@ -224,6 +226,14 @@ export const LocationSuggest = forwardRef<
             tone="critical"
           />
         ) : null}
+
+        {selectedLocation ? (
+          <BreadCrumbsString
```

Comment:
I don't think we want this to display in all usages of location suggest; it likely does not need to be there in an actual posting form or the example on the Developer Site. Can the components be decoupled or can this be an opt-in feature?

Reply:
Added an optional toggle. We could also separate add this to developer dashboard and ryanair but I think this is common enough to add into wingman